dojo/settings/settings.dist.py

```diff
@@ -301,6 +301,8 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
         }
     }
 
+# https://docs.djangoproject.com/en/4.0/ref/settings/#std:setting-CONN_MAX_AGE
+DATABASES['default']['CONN_MAX_AGE'] = 60
```

Comment:
introducing database pooling should probably be in a PR of itself as it's a major thing. at 60s it probably won't do much except for high traffic sites.